test/test_helper.rb

```diff
@@ -258,9 +258,6 @@ def next_mac(mac)
     end
   end
 
-  # Transactional fixtures do not work with Selenium tests, because Capybara
```

Comment:
I thought I changed the line before to 'transaction', I guess I missed it - with that change yeah, it's not accurate